push-build.sh

```diff
@@ -1348,6 +1348,29 @@ release::gcs::publish () {
   fi
 }
 
+ensure_crane_exists () {
+  if ! command -v crane &> /dev/null; then
+    echo "Installing crane..."
+    CRANE_VERSION="v0.19.1"
+    BASE_URL="https://github.com/google/go-containerregistry/releases/download"
+    TAR_NAME="go-containerregistry_Linux_x86_64.tar.gz"
```

Comment:
Are we sure that this script is only used on amd64 systems? cc @kubernetes/release-engineering

Reply:
Done, changed to ``TAR_NAME="go-containerregistry_Linux_$(uname -p).tar.gz"``, which should include the processor type.